sysvar: add v6.5 sysvar (#11569)
pingcap/docs: system-variables.md

- Default value: `ON`
- This variables specifies whether to use the pipeline execution algorithm for window functions.

### tidb_enable_plan_replayer_capture

> Warning:
>
> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.

- Scope: SESSION | GLOBAL

> - Persists to cluster: Yes

Author: ```suggestion
- Persists to cluster: Yes
- Type: Boolean
```

- Default value: `ON`
- This variables specifies whether to use the pipeline execution algorithm for window functions.

### tidb_enable_plan_replayer_capture

> Warning:
>

> > The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.

Reviewer: ```suggestion
> This variable controls a feature that is not fully functional in the current TiDB version. Do not change the default value.
```

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Type: Boolean

> - Default value: `FALSE`

Author: ```suggestion
- Default value: `OFF`
```
